### PR TITLE
Change TranslationTo to TranslatedToLanguageCode

### PR DESCRIPTION
**Pull request by ZulaMostafa**
changed TranslationTo to TranslatedToLanguageCode as was mentioned in the issue [#23350](https://github.com/Azure/azure-sdk-for-net/issues/23350)

### CODE REVIEW

**Review by maririos — COMMENTED** [review]
looks like test and samples need to be updated too

File: sdk/translation/Azure.AI.Translation.Document/tests/samples/Sample_MultipleInputs.cs [hunk 1]
```diff
@@ -95,7 +95,7 @@ public void MultipleInputs()
                 if (document.Status == DocumentTranslationStatus.Succeeded)
                 {
                     Console.WriteLine($"  Translated Document Uri: {document.TranslatedDocumentUri}");
-                    Console.WriteLine($"  Translated to language: {document.TranslatedTo}.");
+                    Console.WriteLine($"  Translated to language: {document.TranslatedToLanguageCode}.");
```
Comment (maririos) [inline sdk/translation/Azure.AI.Translation.Document/tests/samples/Sample_MultipleInputs.cs:98]:
```suggestion
                    Console.WriteLine($"  Translated to language code: {document.TranslatedToLanguageCode}.");
```

This should apply to all samples where this phrase is use

File: sdk/translation/Azure.AI.Translation.Document/tests/samples/Sample_MultipleInputsAsync.cs [hunk 2]
```diff
@@ -1,4 +1,4 @@
-﻿// Copyright (c) Microsoft Corporation. All rights reserved.
+﻿    // Copyright (c) Microsoft Corporation. All rights reserved.
```
Comment (maririos) [inline sdk/translation/Azure.AI.Translation.Document/tests/samples/Sample_MultipleInputsAsync.cs:1]: nit: extra spaces

File: sdk/translation/Azure.AI.Translation.Document/tests/samples/Sample_MultipleInputs.cs (hunk 1 above)
Comment (ZulaMostafa) [reply sdk/translation/Azure.AI.Translation.Document/tests/samples/Sample_MultipleInputs.cs]: Done

File: sdk/translation/Azure.AI.Translation.Document/tests/samples/Sample_MultipleInputsAsync.cs (hunk 2 above)
Comment (ZulaMostafa) [reply sdk/translation/Azure.AI.Translation.Document/tests/samples/Sample_MultipleInputsAsync.cs]: Done

**Review by maririos — APPROVED** [review]

**Pull request merged**